docs/content/environment_tests.md

```diff
@@ -37,15 +37,15 @@ To have a properly reproducible environment that utilizes randomness, you need t
 
 The seed test takes in a function that creates a pettingzoo environment. For example
 
-``` python
+``` python notest
```

Comment:
Reason for this failing?

Reply:
Oh crap, that test was already passing, thanks for catching that